src/mouse_events.hpp

```diff
@@ -87,16 +87,30 @@ class mouse_handler : public mouse_handler_base {
 
 	void select_or_action(bool browse);
 
-	/// Wraps 'const map_location game_display::hex_clicked_on(int, int) const'.
-	const map_location hex_clicked_on(int x, int y) const;
+	/**
+	 * Uses SDL and @ref game_display::hex_clicked_on
+	 * to fetch the hex the mouse is hovering, if applicable.
+	 * @see const game_display::hex_clicked_on(int x, int y) const;
```

Comment:
Please remove the `const` qualifiers from this line, `@see` doesn't work with them.

In addition, specifying the parameters is optional and I recommend against it. Without specifying the parameters, the link will continue to work if the parameters change (if the numbers become unsigned, an additional parameter is added, or whatever).